config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -220,6 +225,9 @@ config MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG
 config SYSTEM_WORKQUEUE_STACK_SIZE
     default 2048
 
+config NET_MGMT_EVENT_STACK_SIZE
```

Comment:
nit: actually not needed because this is exactly what the supplicant Zephyr layer sets by default:
https://github.com/nrfconnect/sdk-hostap/blob/main/zephyr/Kconfig#L90